<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="FTR3COF93BK8A76" name="Default test" tests="26" failures="21" timestamp="29 Mar 2021 03:27:26 GMT" time="6161.221" errors="0">
  <testcase name="IntelliSense_01_Term_Filter_Rex" time="259.598" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [201] line_end; expected [Use] but found [User]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -&amp;gt; xpath: //ul[@id=&amp;apos;AutoCompleteMenu&amp;apos;]/li//strong]] -&amp;gt; xpath: .//span[@style=&amp;apos;color:red;&amp;apos;]] gui_end,
	@@line_start [202] line_end; expected [use] but found [user]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -&amp;gt; xpath: //ul[@id=&amp;apos;AutoCompleteMenu&amp;apos;]/li//strong]] -&amp;gt; xpath: .//span[@style=&amp;apos;color:red;&amp;apos;]] gui_end,
	@@line_start [203] line_end; expected [use] but found [user]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -&amp;gt; xpath: //ul[@id=&amp;apos;AutoCompleteMenu&amp;apos;]/li//strong]] -&amp;gt; xpath: .//span[@style=&amp;apos;color:red;&amp;apos;]] gui_end,
	@@line_start [207] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [208] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [209] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [210] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [211] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [201] line_end; expected [Use] but found [User]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -> xpath: //ul[@id='AutoCompleteMenu']/li//strong]] -> xpath: .//span[@style='color:red;']] gui_end,
	@@line_start [202] line_end; expected [use] but found [user]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -> xpath: //ul[@id='AutoCompleteMenu']/li//strong]] -> xpath: .//span[@style='color:red;']] gui_end,
	@@line_start [203] line_end; expected [use] but found [user]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -> xpath: //ul[@id='AutoCompleteMenu']/li//strong]] -> xpath: .//span[@style='color:red;']] gui_end,
	@@line_start [207] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [208] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [209] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [210] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [211] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_01_Term_Filter_Rex(UI_Data_file_termdef.java:272)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_01_Term_Filter_Rex -->
  <testcase name="IntelliSense_02_Term_Oprations_Rex" time="144.049" classname="testcases.UI_Data_file_termdef"/>
  <testcase name="IntelliSense_03_Term_Triggers_Rex" time="218.056" classname="testcases.UI_Data_file_termdef"/>
  <testcase name="IntelliSense_04_Term_Orange_Rex" time="210.365" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [515] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end,
	@@line_start [516] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end,
	@@line_start [537] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [515] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end,
	@@line_start [516] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end,
	@@line_start [537] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_04_Term_Orange_Rex(UI_Data_file_termdef.java:642)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_04_Term_Orange_Rex -->
  <testcase name="IntelliSense_06_Term_Highlight_SingleMulti_use_Rex" time="97.77" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [788] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [788] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_06_Term_Highlight_SingleMulti_use_Rex(UI_Data_file_termdef.java:808)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_06_Term_Highlight_SingleMulti_use_Rex -->
  <testcase name="IntelliSense_07_Term_Highlight_EditEffects_Rex" time="154.296" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [846] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [847] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [848] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [849] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [846] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [847] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [848] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [849] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_07_Term_Highlight_EditEffects_Rex(UI_Data_file_termdef.java:951)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_07_Term_Highlight_EditEffects_Rex -->
  <testcase name="IntelliSense_08_Term_Replace_Rex" time="196.007" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [986] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [986] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_08_Term_Replace_Rex(UI_Data_file_termdef.java:1169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_08_Term_Replace_Rex -->
  <testcase name="IntelliSense_09_Term_Nest_Rex" time="240.881" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [1192] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1193] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1195] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1197] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1200] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [1192] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1193] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1195] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1197] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1200] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_09_Term_Nest_Rex(UI_Data_file_termdef.java:1301)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_09_Term_Nest_Rex -->
  <testcase name="IntelliSense_10_Term_Filter_Ucx" time="175.939" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [1344] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1345] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1375] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [1344] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1345] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1375] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_10_Term_Filter_Ucx(UI_Data_file_termdef.java:1419)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_10_Term_Filter_Ucx -->
  <testcase name="IntelliSense_11_Term_Operations_Ucx" time="239.898" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [1455] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1456] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1457] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1458] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1459] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1462] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [1455] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1456] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1457] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1458] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1459] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1462] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_11_Term_Operations_Ucx(UI_Data_file_termdef.java:1554)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_11_Term_Operations_Ucx -->
  <testcase name="IntelliSense_12_Term_Triggers_Ucx" time="145.669" classname="testcases.UI_Data_file_termdef"/>
  <testcase name="IntelliSense_13_Term_Orange_Ucx" time="172.802" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [1688] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1689] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1703] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [1688] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1689] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [1703] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_13_Term_Orange_Ucx(UI_Data_file_termdef.java:1768)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_13_Term_Orange_Ucx -->
  <testcase name="IntelliSense_14_Term_Highlight_Realtime_Ucx" time="543.168" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [1785] line_end; expected [rgba(0, 128, 0, 1)] but found [rgba(255, 0, 0, 1)],
	@@line_start [1786] line_end; expected [rgba(255, 165, 0, 1)] but found [rgba(0, 128, 0, 1)],
	@@line_start [1803] line_end; expected [Not found] but found [Training cyberobject/uitest_cyberobject_com_cn/intellisensetest/data/IntelliSense/termhighlight_domain.xlsx success !]gui_start  gui_end,
	@@line_start [1832] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [1785] line_end; expected [rgba(0, 128, 0, 1)] but found [rgba(255, 0, 0, 1)],
	@@line_start [1786] line_end; expected [rgba(255, 165, 0, 1)] but found [rgba(0, 128, 0, 1)],
	@@line_start [1803] line_end; expected [Not found] but found [Training cyberobject/uitest_cyberobject_com_cn/intellisensetest/data/IntelliSense/termhighlight_domain.xlsx success !]gui_start  gui_end,
	@@line_start [1832] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_14_Term_Highlight_Realtime_Ucx(UI_Data_file_termdef.java:1910)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_14_Term_Highlight_Realtime_Ucx -->
  <testcase name="IntelliSense_15_Term_Highlight_Single_Multi_use_Ucx" time="170.269" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [1973] line_end; expected [rgba(0, 0, 0, 1)] but found [rgba(255, 0, 0, 1)]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [1973] line_end; expected [rgba(0, 0, 0, 1)] but found [rgba(255, 0, 0, 1)]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_15_Term_Highlight_Single_Multi_use_Ucx(UI_Data_file_termdef.java:2001)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_15_Term_Highlight_Single_Multi_use_Ucx -->
  <testcase name="IntelliSense_17_Term_Replace_Ucx" time="237.001" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2164] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2164] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_17_Term_Replace_Ucx(UI_Data_file_termdef.java:2249)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_17_Term_Replace_Ucx -->
  <testcase name="IntelliSense_18_Term_Nest_Ucx" time="222.599" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2279] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2279] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_18_Term_Nest_Ucx(UI_Data_file_termdef.java:2337)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_18_Term_Nest_Ucx -->
  <testcase name="IntelliSense_32_Term_TopLevelTerm_TableTerm" time="284.712" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2397] line_end; expected [NotFound] but found [Displayed]gui_start By.xpath: //ul[@id=&amp;apos;AutoCompleteMenu&amp;apos; and contains(@style,&amp;apos;display: block&amp;apos;)] gui_end,
	@@line_start [2401] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2410] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2411] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2412] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2432] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2433] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2456] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2465] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2466] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2397] line_end; expected [NotFound] but found [Displayed]gui_start By.xpath: //ul[@id='AutoCompleteMenu' and contains(@style,'display: block')] gui_end,
	@@line_start [2401] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2410] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2411] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2412] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2432] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2433] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2456] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2465] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2466] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_32_Term_TopLevelTerm_TableTerm(UI_Data_file_termdef.java:2491)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_32_Term_TopLevelTerm_TableTerm -->
  <testcase name="IntelliSense_33_Resource_Filter_Rex" time="109.285" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2554] line_end; expected [File/User ] but found [Fi]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -&amp;gt; xpath: //ul[@id=&amp;apos;AutoCompleteMenu&amp;apos;]/li[1]//strong]] -&amp;gt; xpath: .//span[@style=&amp;apos;color:red;&amp;apos;]] gui_end,
	@@line_start [2562] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2563] line_end; expected [File/User] but found [File/User ]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -&amp;gt; xpath: //ul[@id=&amp;apos;AutoCompleteMenu&amp;apos;]/li//strong]] -&amp;gt; xpath: .//span[@style=&amp;apos;color:red;&amp;apos;]] gui_end,
	@@line_start [2564] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end,
	@@line_start [2571] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2554] line_end; expected [File/User ] but found [Fi]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -> xpath: //ul[@id='AutoCompleteMenu']/li[1]//strong]] -> xpath: .//span[@style='color:red;']] gui_end,
	@@line_start [2562] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2563] line_end; expected [File/User] but found [File/User ]gui_start [[[[ChromeDriver: chrome on XP (aa8b00c2be41d7ad3ed8a045461ba8b8)] -> xpath: //ul[@id='AutoCompleteMenu']/li//strong]] -> xpath: .//span[@style='color:red;']] gui_end,
	@@line_start [2564] line_end; expected [Displayed] but found [NoSuchElement]gui_start element is null gui_end,
	@@line_start [2571] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_33_Resource_Filter_Rex(UI_Data_file_termdef.java:2583)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_33_Resource_Filter_Rex -->
  <testcase name="IntelliSense_34_Resource_Operations_Rex" time="127.424" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2668] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2669] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2670] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2668] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2669] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2670] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_34_Resource_Operations_Rex(UI_Data_file_termdef.java:2698)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_34_Resource_Operations_Rex -->
  <testcase name="IntelliSense_35_Resource_Triggers_Rex" time="169.5" classname="testcases.UI_Data_file_termdef"/>
  <testcase name="IntelliSense_36_Resource_Highlight_Realtime_Rex" time="519.108" classname="testcases.UI_Data_file_termdef"/>
  <testcase name="IntelliSense_37_Resource_Highlight_EditEffects_Rex" time="151.18" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2989] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2990] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2989] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [2990] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_37_Resource_Highlight_EditEffects_Rex(UI_Data_file_termdef.java:3034)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_37_Resource_Highlight_EditEffects_Rex -->
  <testcase name="IntelliSense_38_Resource_Filter_Ucx" time="204.967" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [3071] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3072] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3073] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3074] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3121] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3122] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3123] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3124] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3138] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [3071] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3072] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3073] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3074] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3121] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3122] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3123] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3124] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3138] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_38_Resource_Filter_Ucx(UI_Data_file_termdef.java:3148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_38_Resource_Filter_Ucx -->
  <testcase name="IntelliSense_39_Resource_Operations_Ucx" time="157.962" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [3201] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3233] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [3201] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3233] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_39_Resource_Operations_Ucx(UI_Data_file_termdef.java:3301)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_39_Resource_Operations_Ucx -->
  <testcase name="IntelliSense_40_Resource_Triggers_Ucx" time="288.032" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [3345] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3346] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3347] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3348] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3354] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3366] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3367] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3368] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3380] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3386] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [3345] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3346] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3347] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3348] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3354] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3366] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3367] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3368] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3380] line_end; expected [NotFound] but found [Displayed]gui_start  gui_end,
	@@line_start [3386] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_40_Resource_Triggers_Ucx(UI_Data_file_termdef.java:3397)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_40_Resource_Triggers_Ucx -->
  <testcase name="IntelliSense_41_Resource_Highlight_Realtime_Ucx" time="128.104" classname="testcases.UI_Data_file_termdef">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	@@line_start [2557] line_end; expected [] but found [MouseRightClick failed!]gui_start  gui_end,
	@@line_start [3404] line_end;gui_start java.lang.NullPointerException gui_end">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	@@line_start [2557] line_end; expected [] but found [MouseRightClick failed!]gui_start  gui_end,
	@@line_start [3404] line_end;gui_start java.lang.NullPointerException gui_end
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at testcases.UI_Data_file_termdef.IntelliSense_41_Resource_Highlight_Realtime_Ucx(UI_Data_file_termdef.java:3423)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- IntelliSense_41_Resource_Highlight_Realtime_Ucx -->
</testsuite> <!-- Default test -->
